mechanics/tools/gdocs-migrate/forest.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+type File struct {
+	id       string
+	name     string
+	parentId string
+	isFolder bool
+}
+
+type Forest struct {
+	self     map[string]File
```

Comment:
why self is a map? one would expect self is a single entry.

Reply:
I'll rename to `fileById`

Reply:
Done